### DISCUSSION TITLE
BelongTo with Custom TItle

### DISCUSSION THREAD

**Discussion by monaye** [laravel/nova-issues#4359] [Q&A] [answered]
Currently, `BelongsTo` field displays the title/subtitle defined in the Resources, however, on some occurrences, would like to display the custom title/subtitle. I believe this is not possible at this moment unless I use the custom HTML link.

**Comment by philipbaginski** [comment]
Hi!

If I understand correctly you can make it like:

BelongsTo::make(__('Player'), 'player', Player::class)
                        ->display(function ($request) {
                            return $request->player_name; // player_name set up in model file.
                        }),
or:

BelongsTo::make(__('Player'), 'player', Player::class)
                        ->label(__('Player')),  // label set up in resource file.
or:

BelongsTo::make(__('Player'), 'player', Player::class)
                        ->singularLabel(__('Player')), // singularLabel set up in resource file.

Let me know if it helps.

**Comment by monaye** [comment, marked as answer]
@philipbaginski thank you :) I don't know how I missed it... it was right in front of my face...  looks like `displayUsing` is proxy for the `display` and call back function receives the  Nova `$resource` 

```
->displayUsing(function ($resource) {
                    if (request()->user()->isOwner()) {
                        return $resource->id . '-' . $resource->name;
                    }
                    return $resource->name;
                })
```
